src/_common.scss

```diff
@@ -1,6 +1,7 @@
 @import '~box-react-ui/lib/styles/variables';
 
-.ba-annotation-dialog {
+.ba-annotation-dialog,
```

Comment:
Are there any different classes for mobile?

Reply:
i'll confirm but they should still have the `ba-annotation-dialog` selector